### ISSUE TITLE
Issue with Mongodb JPQL LIKE queries

### ISSUE THREAD

**Issue by arvindNeergunda** [Impetus/kundera#611]
I was trying to get the below query working and it does not seem to work.
Query query = em.createQuery( "SELECT c FROM School c WHERE c.city LIKE :city " ) ;
query.setMaxResults( 10 );
query.setParameter( "city", "Ban%" );

The % query fails. Also When i use SELECT c FROM School c WHERE  ( ( c.city LIKE :city ) AND (c.area = :area) )

Are multiple parenthesis in query have issues in Kundera. 
Are there any work being done to support complex JPQL queries ?

thanks
Arvind

**Comment by chhavigangwal** [comment]
Complex JPQL queries with multiple brackets are not supported in Kundera as of now, but will be planned for future releases.

Support for % in like query will be added in 2.13 release. For now you can try using like query by changing 

```
query.setParameter( "city", "Ban%" );
```

to

```
query.setParameter( "city", "Ban*" );
```

and where query 

```
SELECT c FROM School c WHERE  ( c.city LIKE :city ) AND (c.area = :area) 
```

Chhavi

**Comment by chhavigangwal** [comment]
Fixed releasing with 2.13
